### ISSUE TITLE
modularscale-sass needed in deps?

### ISSUE THREAD

**Issue by ungoldman** [ungoldman/style.css#27]
```
> ungoldman.com@1.0.0-beta sass /Users/ngoldman/dev/git/ungoldman.com
> node-sass source/scss/ -o build/css/ --include-path node_modules/

{
  "status": 1,
  "file": "/Users/ngoldman/dev/git/ungoldman.com/node_modules/style.css/style.scss",
  "line": 3,
  "column": 1,
  "message": "File to import not found or unreadable: modularscale-sass/stylesheets/modular-scale\nParent style sheet: /Users/ngoldman/dev/git/ungoldman.com/node_modules/style.css/style.scss",
  "formatted": "Error: File to import not found or unreadable: modularscale-sass/stylesheets/modular-scale\n       Parent style sheet: /Users/ngoldman/dev/git/ungoldman.com/node_modules/style.css/style.scss\n        on line 3 of node_modules/style.css/style.scss\n>> @import 'modularscale-sass/stylesheets/modular-scale';\n   ^\n"
}
```

💩

**Comment by ungoldman** [comment]
yep, tested in https://github.com/ungoldman/ungoldman.com/commit/9bafc5d0ac0414ba7c9daeea74e8261449d2e2bb

**Comment by ungoldman** [comment]
fixed in 1.0.0-alpha-7
